eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -198,3 +201,18 @@ services:
 
     ez_query:
         alias: ezpublish.controller.query
+
+    ezpublish.signalslot.signal_dispatcher_transaction_wrapper:
+        class: %ezpublish.signalslot.signal_dispatcher_transaction_wrapper.class%
+        arguments: [@ezpublish.signalslot.signal_dispatcher]
+
+    ezpublish.signalslot.signal_dispatcher:
+        class: %ezpublish.signalslot.signal_dispatcher.class%
+        factory: ["@ezpublish.signalslot.signal_dispatcher.factory", buildSignalDispatcher]
+
+    ezpublish.signalslot.signal_dispatcher.factory:
```

Comment:
You should rather place this in `eZ/Bundle/EzPublishCoreBundle/Resources/config/papi.yml` to achieve override.